tests/regression/56-floats/03-infinity_or_nan.c

```diff
@@ -0,0 +1,11 @@
+// PARAM: --enable ana.float.interval
```

Comment:
you will probably have to enable the now by default disabled float warnings for this test

Reply:
I agree. after that it seems good to merge

Reply:
Great catch, thank you! This is also done now